websites/F/Flat/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "Flat",
+  "author": {
+    "id": "969650307464298636",
+    "name": "PiggyPlex#0772"
```

Comment:
```suggestion
    "name": "PiggyPlex"
```

Reply:
The CLI automatically used that former format - can I get confirmation that the latter is the preferred method?

Reply:
The CLI wouldn't have added your user discriminator

Reply:
hmmn, it asked for my user ID and filled it all in automatically - I didn't edit any of it